core/src/test/scala/unit/kafka/metrics/MetricsTest.scala

```diff
@@ -158,7 +159,7 @@ class MetricsTest extends KafkaServerTestHarness with Logging {
     val path = "C:\\windows-path\\kafka-logs"
     val tags = Map("dir" -> path)
     val expectedMBeanName = Set(tags.keySet.head, ObjectName.quote(path)).mkString("=")
-    val metric = KafkaMetricsGroup.metricName("test-metric", tags)
+    val metric = new KafkaMetricsGroup(this.getClass).metricName("test-metric", tags.asJava)
```

Comment:
Worth noting that this is slightly different than the old code, but seems ok for what we're testing here.